ui/src/components/directory/viewer/graph/GraphComponent.vue

```diff
@@ -69,7 +69,7 @@ const width = ref(400);
 const force = ref(-5000);
 const radius = ref(16);
 const colour = ref({
-  activeNode: { fill: "var(--background-a)", stroke: "#AAAAAA" },
+  activeNode: { fill: "#FFFFFF", stroke: "#AAAAAA" },
```

Comment:
Are there any appropriate variables we can use for this, rather than hard-coded #FFFFFF (and also the stroke/#AAAAAA)?